bft/testers/cudf/runner.py

```diff
@@ -22,6 +23,12 @@ def type_to_cudf_dtype(type: str):
     return type_map[type]
 
 
+def literal_to_str(lit):
```

Comment:
Do we want to start using type annotations in Python?

Reply:
This function actually isn't being used anymore.  Removing it.  Thanks for the catch!